docs/general/mutations.md

```diff
@@ -4,12 +4,13 @@ title: Mutations
 
 # Mutations
 
-As opposed to queries, mutation in GraphQL represent operation that cause writes
-and/or side effects on the server. For example you can have a mutation that
-sends an email or a mutation that creates a user.
+As opposed to queries, mutations in GraphQL represent operations that modify
+server-side data and/or cause side effects on the server. For example, you can have a
+mutation that creates a user or a mutation that sends an email. Like in queries, they
+accept parameters, can return data or include nested fields as well.
```

Comment:
```
or include nested fields as well.
```
what do you mean by this? 😊

Reply:
That like queries you can write 
```
  {
    data {
       product {
          title
          reviews {
          ...
          }
      }
   }
```
Maybe is something obvious... 
is something the [GraphQL docs was mention 
](https://graphql.org/learn/queries/#mutations) so I thought to put something similar 
> Just like in queries, if the mutation field returns an object type, you can ask for nested fields.